src/components/molecules/close-contact-warning.tsx

```diff
@@ -49,7 +51,7 @@ const styles = StyleSheet.create({
   notice: {
     ...text.defaultBold,
     color: colors.white,
-    paddingRight: 35,
-    paddingLeft: 10
+    paddingHorizontal: 16,
```

Comment:
What used to happen and what did it change to?

Reply:
Will re-assess as above